Source/WebInspectorUI/UserInterface/Views/GeneralStyleDetailsSidebarPanel.js

```diff
@@ -193,13 +164,25 @@ WI.GeneralStyleDetailsSidebarPanel = class GeneralStyleDetailsSidebarPanel exten
         }
 
         if (this._classListContainer) {
-            this._classToggleButton = optionsContainer.createChild("button", "toggle-class-toggle");
-            this._classToggleButton.textContent = WI.UIString("Classes");
-            this._classToggleButton.title = WI.UIString("Toggle Classes");
-            this._classToggleButton.addEventListener("click", this._classToggleButtonClicked.bind(this));
+            this._classListToggleButton = optionsContainer.createChild("button", "toggle");
+            this._classListToggleButton.textContent = WI.UIString("Classes");
+            this._classListToggleButton.title = WI.UIString("Toggle Classes");
+            this._classListToggleButton.addEventListener("click", this._classListToggleButtonClicked.bind(this));
+
+            this._updateClassListContainer();
+        }
+
+        if (this._forcedPseudoClassContainer) {
+            this._forcedPseudoClassToggleButton = optionsContainer.appendChild(document.createElement("button"));
+            this._forcedPseudoClassToggleButton.className = "toggle";
+            this._forcedPseudoClassToggleButton.textContent = WI.UIString("Pseudo");
+            this._forcedPseudoClassToggleButton.title = WI.UIString("Toggle Pseudo Classes");
+            this._forcedPseudoClassToggleButton.addEventListener("click", this._forcedPseudoClassToggleButtonClicked.bind(this));
 
-            if (this._classListContainerToggledSetting.value)
-                this._classToggleButtonClicked();
+            if (this.exclusive && this._classListContainerToggledSetting.value === this._classListContainerToggledSetting.defaultValue && this._forcedPseudoClassContainerToggledSetting.value === this._forcedPseudoClassContainerToggledSetting.defaultValue)
+                this._forcedPseudoClassContainerToggledSetting.value = true;
```

Comment:
I like the idea here, but it seems like this means if I'm resizing Web Inspector, we collapse the sidebars together, and then I uncollapse them it is probable that a previously closed pseudo-classes container will be shown again, which sounds kinda annoying.

Reply:
so the answer to this is "yes" and "no".  this is happening inside `initialLayout`, so it should only be done once.  but this is subclassed, so each of the subclasses will also attempt to do this, so we might wanna restrict this to just the Styles details sidebar panel.

also, this will stay expanded even if we collapse the sidebars together, since it's ultimately the same `WI.SidebarPanel` instance

we probably should make these into global `WI.settings` tho so that they don't clash with depending on which details sidebar panel is shown